docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -33,3 +33,14 @@ If you want to perform development tasks inside the {project} VM, it is recommen
 Depending on your operating system and shell environment, certain special characters can trigger variable interpolation and therefore cause passwords to fail.
 
 Workaround: When creating and entering passwords, wrap the string with single quotes in the following format: `'<password>'`
+
+
+[[authentication-required-to-push-image]]
+== Authentication required to push image to OpenShift integrated Docker registry
+
+If you are using Fedora/CentOS/RHEL host and you are running {project} with the Boot2Docker or Minikube xref:../using/basic-usage.adoc#choosing-iso-image#[ISO] 
+then even after following the registry xref:../openshift/openshift-docker-registry.adoc#[documents], 
+you might see an authorization error when trying to push an image to exposed registry 
+because of incompatibility between the docker client shipped with Fedora/CentOS/RHEL and docker daemon which is running inside these ISO.
```

Comment:
Docker not docker

Reply:
The Docker daemon is not really running inside the ISO. Inside the {project} VM might be better.

Reply:
what are the incompatibilities? Do we have a link to an issue or something?

Reply:
> what are the incompatibilities? Do we have a link to an issue or something?

Unfortunately, we don't have an issue on Bugzilla and we also can't create it because this is not something broken in the fedora, centos or rhel side (you can use docker client and daemon perfectly over these distros when trying to push/pull images). Problem is we are using an unsupported docker daemon from the fedora/centos/rhel ecosystem when running b2d or minikube ISO and they are not responsible to provide any fix for same.